sdk/core/azure-core/src/main/java/com/azure/core/util/serializer/jsonwrapper/api/Node.java

```diff
@@ -1,9 +1,12 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Do we need to make these classes public? Only public the necessary one.

Reply:
jacksonwrapper.JacksonNode is implemeting Node.